core/src/cljfmt/format/fn.clj

```diff
@@ -68,8 +67,8 @@
          (zl/token? unwrapped)
          (no-prev? zloc arg-vector?)
          (symbol? (z/sexpr unwrapped))
-         (not (contains? #{"fn" "defn"} (name (z/sexpr unwrapped))))
-         (contains? #{[] ["fn"] ["defn"]} (map name (preceeding-symbols zloc))))))
+         (not (contains? #{"fn" "defn" "defn-"} (name (z/sexpr unwrapped))))
+         (contains? #{[] ["fn"] ["defn"] ["defn-"]} (map name (preceeding-symbols zloc))))))
```

Comment:
This could also use the refactored predicate, something like:
```clojure
(let [preceeding (preceeding-symbols zloc)]
  (or (empty? preceeding)
      (and (= 1 (count preceeding))
           (fn-sym? (first preceeding)))))
```